css/properties/flex-direction.json

```diff
@@ -0,0 +1,85 @@
+{
+  "css": {
+    "properties": {
+      "flex-direction": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/flex-direction",
+          "support": {
+            "chrome": {
+              "prefix": "-webkit-",
+              "version_added": "21"
+            },
+            "edge": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "edge_mobile": [
+              {
+                "version_added": true
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "20",
+                "notes": "Since Firefox 28, multi-line flexbox is supported."
+              },
+              {
+                "version_added": "18",
+                "version_removed": "20",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.flexbox.enabled",
+                  "value_to_set": "true"
+                }
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "49",
+                "notes": "For Firefox 48, the <code>-webkit-</code> prefix was available with the <code>layout.css.prefixes.webkit</code> preference. Starting with Firefox 49, the preference defaults to <code>true</code>."
+              }
+            ],
+            "ie": [
+              {
+                "version_added": "11"
+              },
+              {
+                "prefix": "-ms-",
+                "version_added": "10"
+              }
+            ],
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "12.10"
+            },
+            "opera_android": {
+              "version_added": "12.10"
+            },
+            "safari": {
+              "prefix": "-webkit-",
+              "version_added": "7"
+            },
+            "safari_ios": {
+              "version_added": false
```

Comment:
This looks dubious. Maybe 'null' instead.

Reply:
Done.